### ISSUE TITLE
Add source for legos components

### ISSUE THREAD

**Issue by bluewaysw** [bluewaysw/pcgeos#596]

**Comment by HubertHuckevoll** [comment]
It would also be cool to have the source code of the rest of NewBasic in the repo. The "Builder" could be used as the starting point for a visual UI designer that exports GOC/UI code...

**Comment by bluewaysw** [comment]
Yes, this is the plan. I think all of this is project "Legos" in it origin.

**Comment by HubertHuckevoll** [comment]
Do you have the full source code? - if not, I might have parts of it...

blueway.Softworks ***@***.***> schrieb am Fr., 12. Juli 2024,
16:09:

**Comment by nschulux** [comment]
The "Builder" could be used as the starting point for a visual UI designer for Rainers R-Basic.
